### PR TITLE
Career stats

### PR DESCRIPTION
**Pull request by johnwheffner**

### CODE REVIEW

**Comment by johnwheffner** [comment]
One question: should leader boards be limited to a fixed number of racers?  It's kind of nice to be able to see all of them.  Pagination might be a good thing, but I don't feel like doing that now.  (We'd want to js-ify everything to do that, with multiple tables on one page.)

**Comment by jyundt** [comment]
> One question: should leader boards be limited to a fixed number of racers?

For now: I would say "no", let's just go with a long scroll + all data.  I like your idea of JS + pagination at some point, but I think the current view is fine.

A few ideas:

- Would you be amicable to changing the date format from `YYYY-MM-DD` to `MM/DD/YYYY` for consistency?
- Do you think "stats" is descriptive enough?  Maybe something like "all-time leaderboard" or something?  
- Thoughts on having some sort of on-page nav to link between "Career Points", "Career MAR Points", "Career Wins" and "Trifectas"? Or if not a nav, some other way to display these multiple headers at the top of the page?  I didn't realize there were additional categories (e.g. "Career Wins") until I started scrolling down.

**Comment by johnwheffner** [comment]
(1) Yes, will do.
(2) I was looking for something better than "Stats" but couldn't find anything that wasn't too narrow.  Open to suggestions.
(3) Good point.  I'll try a couple things -- at least add some anchors and links at the top, maybe doing a show/hide of the divs.

**Comment by jyundt** [comment]
> (2) I was looking for something better than "Stats" but couldn't find anything that wasn't too narrow. Open to suggestions.

"Overall racer stats", "Career stats"?  I dunno, maybe I'll ask @emilywools, she has good words, the *best* words.

> (3) Good point. I'll try a couple things -- at least add some anchors and links at the top, maybe doing a show/hide of the divs.

I'm not 100% familiar with all of the widgets in bootstrap, but maybe something like [collapse.js](http://getbootstrap.com/javascript/#collapse)?

**Comment by jyundt** [comment]
Semi-related: I think I need to downgrade to reclaim the title of "Most B MAR points":

image

**Comment by johnwheffner** [comment]
Added accordion panels.  Not sure how I like them.  Give it a try.
I might prefer paginated tables.

File: app/templates/stats/index.html [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+          </tbody>
+        </table>
+      </div>
+    </div>
+  </div>
+</div>
+{% endif %}
+
+<div>
+  Note: statistics displayed are since {{earliest_date}}.
```
Comment (jyundt) [inline app/templates/stats/index.html]: Would you be amicable to changing this to mm/dd/yyyy as well?

**Comment by jyundt** [comment]
I kinda like the accordion. Or if that doesn't do it for you, maybe some other type of ToC/header/sub-nav for "Career points, Career MAR points, etc...."

**Comment by johnwheffner** [comment]
The thing I mainly don't like about the accordion is that it doesn't seem obvious I'm supposed to click to expand.  Could also try some plain old links and anchors.

**Comment by emilywools** [comment]
can you screenshot me or link me to what you are talking about?

**Comment by jyundt** [comment]
image

image


image

**Comment by emilywools** [comment]
i think it would be most clear to include a + and a - at the right side of
the accordion item to ensure people know that its clickable and it helps
inform the anticipated behavior when clicked.

**Comment by johnwheffner** [comment]
Maybe an up/down (glyphicon-menu-up).  Would require some custom javascript
I think.

**Comment by jyundt** [comment]
What about some other dropdown on the navbar to only display "Trifectas" or "Career Points", etc?

**Comment by johnwheffner** [comment]
That's a possibility.  Would need a lot more templates.

**Comment by johnwheffner** [comment]
I might get time to mess with accordions and drop-downs in a while but I don't think I'll get to it soon.  I'm inclined to just ship this as "good enough for now."

**Review by jyundt — APPROVED** [review]
Agreed, let's ship this as/is and get fancier later.

**Pull request merged**